tools/online_editor/src/preview_widget.ts

```diff
@@ -122,7 +122,14 @@ export class PreviewWidget extends Widget {
       if (this.#instance == null) {
         this.#instance = component.create(this.canvas_id);
         this.#instance.show();
-        slint.run_event_loop();
+        try {
+          slint.run_event_loop();
+        } catch (e) {
```

Comment:
is there a way to check that this exception is really the winit one that is safe to catch?
i guess it might be fragile anyway so i guess that's fine like this.

Reply:
I’ve come to the same conclusion.